jsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=759670
+	// for the documentation about the jsconfig.json format
```

Comment:
Native json file does not support comments. Will be a parse error during parsing file somewhere programmatically